langchain-core/src/runnables/base.ts

```diff
@@ -540,6 +540,31 @@ export abstract class Runnable<
     });
   }
 
+  /**
+   * Pick keys from the dict output of this runnable. Returns a new runnable.
+   */
+  pick(keys: string | string[]): RunnableSequence {
+    // eslint-disable-next-line @typescript-eslint/no-use-before-define, @typescript-eslint/no-explicit-any
+    return this.pipe(new RunnablePick(keys) as any);
```

Comment:
Do we need the cast? And if we do, `as Runnable` is better?

Reply:
we need because of the dict input restriction